src/performance_profiles.jl

```diff
@@ -115,3 +115,22 @@ end
 
 performance_profile(T :: Array{Tn,2}, labels :: Vector{S}; kwargs...) where {Tn <: Number, S <: AbstractString} =
   performance_profile(convert(Array{Float64,2}, T), convert(Vector{AbstractString}, labels); kwargs...)
+
+"""
+    performance_profile(stats, cost)
+Produce a performance profile comparing solvers in `stats` using the `cost` function.
+Inputs:
+- `stats::Dict{Symbol,DataFrame}`: pairs of `:solver => df`;
+- `cost::Function`: cost function applyed to each `df`. Should return a vector with the cost of solving the problem at each row;
+  - 0 cost is not allowed;
+  - If the solver did not solve the problem, return Inf or a negative number.
+Examples of cost functions:
+- `cost(df) = df.elapsed_time`: Simple `elapsed_time` cost. Assumes the solver solved the problem.
+- `cost(df) = (df.status .!= :first_order) * Inf + df.elapsed_time`: Takes into consideration the status of the solver.
+"""
+function performance_profile(stats::Dict{Symbol,DataFrame}, cost::Function, args...; kwargs...)
```

Comment:
```suggestion
function performance_profile(stats::Dict{Symbol,DataFrame}, cost, args...; kwargs...)
```

I suggest to not type `cost` as any callable object should be accepted.

Reply:
Done